Functional tests compiling errors when running against target floatSanityTests.

Functional tests compiling errors when running against target floatSanityTests, which is noticed when running PR tests. Nightly build don't have this issue.

https://github.com/AdoptOpenJDK/run-aqa/actions/runs/385922131

```
    [javac] /home/runner/work/run-aqa/run-aqa/openjdk-tests/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestOpenJ9DiagnosticsMXBean.java:258: error: cannot find symbol
    [javac] 		String[] resetOptions = diagBean.queryDumpOptions();
    [javac] 		                                ^
    [javac]   symbol:   method queryDumpOptions()
    [javac]   location: variable diagBean of type OpenJ9DiagnosticsMXBean

```

Functional tests compiling errors when running against target floatSanityTests.

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -3,7 +3,8 @@ on:
   push:
     branches:
       - master
-
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
 jobs:
   openjdk: # make sure build/ci work properly
     runs-on: ${{ matrix.os }}
@@ -33,6 +34,7 @@ jobs:
         version: '11'
         targets: 'JDK_11'
         impl: 'openj9'
+        source: 'nightly'
     - name: AQA
       uses: ./
       with: 
@@ -53,6 +55,7 @@ jobs:
         version: '11'
         targets: 'JDK_11'
         impl: 'openj9'
+        source: 'nightly'
     - name: AQA
       uses: ./
       with: 

